### PR TITLE
LPS-184338 The url should be based on the object definition scope

### PR DESCRIPTION
**Pull request by seltong**
Related issue: [LPS-184338](https://liferay.atlassian.net/browse/LPS-184338)

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:sf

**Review by murilostodolni — APPROVED** [review]
LGTM 🎉

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java [hunk 1]
```diff
@@ -94,6 +96,15 @@ public Map<String, Object> getProperties(
...
 
 		return HashMapBuilder.<String, Object>put(
+			"objectDefinitionScope",
+			() -> {
+				ObjectDefinition objectDefinition =
+					_objectDefinitionLocalService.getObjectDefinition(
+						objectField.getObjectDefinitionId());
+
+				return objectDefinition.getScope();
+			}
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java:106]: @seltong can we change to use the `objectScopeProvider.isGroupAware()`? Renaming the property to `groupAware`.

Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java (hunk 1 above)
Comment (seltong) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java]: Yeah, we can use it!

Thanks!

**Comment by seltong** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]

**Comment by seltong** [comment]
ci:test:object

**Comment by seltong** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**